order.py

```diff
@@ -0,0 +1,216 @@
+            if order.side == "buy":
+                price_loc = self.BuyPrices.index(order.price)
+                order_loc = self.orderBuys[price_loc].index(order_id)
+                self.orderBuys[price_loc].pop(order_loc)
+                # remove price and the element in orderBuy if it become empty
+                if len(self.orderBuys[price_loc]) == 0:
+                    self.orderBuys.pop[price_loc]
+                    self.BuyPrices.pop[price_loc]
+                # Find where the order is and remove in standing buys
+                self.standingBuys.pop(self.standingBuys.index(order_id))
+                
+            else: # order.side == "sell"
+                price_loc = self.SellPrices.index(order.price)
+                order_loc = self.SellPrices[price_loc].index(order_id)
+                self.orderSellsprice_loc.pop[order_loc]
+                # remove price and the element in orderBuy if it become empty
+                if len(self.orderSells[price_loc]) == 0:
+                    self.orderSells.pop[price_loc]
+                    self.SellPrices.pop[price_loc]
+            cancelStatus = "Order " + str(order_id) + " has been canceled."
+        else:
+            cancelStatus = "Order " + str(order_id) + " cannot be canceled."
+        return(cancelStatus)
+
+    def orderAdd(self, order):
+        new_orderId = self.create_orderId()
+        print("created order id is " + str(new_orderId))
+        order = Order(order.trader, order.side, order.price, order.qty, new_orderId)
+        
+        if order.side == "buy":
+            self.orderMap[order.orderId] = order
+            self.standingBuys.append(order.orderId) # update standing order for buys
+                
+            # If price exist, add to the current index
+            if order.price in self.BuyPrices:
+                self.orderBuys[self.BuyPrices.index(order.price)].append(order.orderId)
+            else: # new price
+                price_ind = bisect.bisect(self.BuyPrices, order.price)
+                self.BuyPrices.insert(price_ind, order.price)
+                self.orderBuys.insert(price_ind, [order.orderId])
+                    
+        else: # order.side == "sell"
+            self.orderMap[order.orderId] = order
+            if order.price in self.SellPrices:
+                self.orderSells[self.SellPrices.index(order.price)].append(order.orderId)
+            else:
+                price_ind = bisect.bisect(self.SellPrices, order.price)
+                self.SellPrices.insert(price_ind, order.price)
+                self.orderSells.insert(price_ind, [order.orderId])
+        
+        return(self.orderProcess(order))
+          
+    def makeTrade(self, buy_order, sell_order):
+        fill_qty = min(buy_order.qty, sell_order.qty)
+        buy_order.qty -= fill_qty
+        sell_order.qty -= fill_qty
+        fill_id = self.new_fillId()
+        fill_timestamp = time.strftime("%Y-%m-%d %H:%M:%S", time.gmtime(time.time()))

```

Comment:
Generally it makes more sense to just save `time.time()` as your timestamp here and only convert it to a string when needing to display it to a user.